src/libraries/filcrypto/filproofs/algorithms.go

```diff
@@ -864,7 +864,10 @@ func (sdr *WinStackedDRG_I) _getChallengedSectors(randomness sector.PoStRandomne
 	panic("TODO")
 }
 
-func (sdr *WinStackedDRG_I) GeneratePoStCandidates(challengeSeed sector.PoStRandomness, eligibleSectors []sector.SectorNumber, candidateCount int, sectorStore sectorIndex.SectorStore) []sector.ChallengeTicket {
+////////////////////////////////////////////////////////////////////////////////
+// Election PoSt
+
+func (sdr *WinStackedDRG_I) GenerateElectionPoStCandidates(challengeSeed sector.PoStRandomness, eligibleSectors []sector.SectorNumber, candidateCount int, sectorStore sectorIndex.SectorStore) []sector.PoStCandidate {
```

Comment:
is `eligibleSectors` the same as `ProvingSet`? Let's stick to the same naming?

Reply:
mmh that is a good point. CC @porcuquine lord of filproofs